### DISCUSSION TITLE
i'm gonna protest till update

### DISCUSSION THREAD

**Discussion by idontveapc** [bromite/bromite#2632] [Builds]
.

and no update means severe warmth in the dev's house.

**Comment by secuencias** [comment]
At this point I think the main developer may be deceased.......hope of course, that is not the case!!!

**Reply by hank-tu-biz** [reply]
I'm in the same boat. I'm using Iceraven until/unless something happens with Bromite.

**Reply by spaha-bih** [reply]
It would be good to know at least is it the project abandoned totally or the dev is taking a longer brake. 

Is there anything else than can replace look and feel of Bromite?

**Reply by kitkat444** [reply]
https://github.com/uazo/cromite

**Comment by hank-tu-biz** [comment]
I found a privacy browser called Iceraven. I like it better than Bromite. Also found was a FOSS app called FFUpdater. This app tracks and updates various privacy apps, including DDG, Iceraven, Tor, Brave, and Orbot plus others. It providers the latest right from GitHub.

You might check these out. They made thing easy for me.

Jul 19, 2023 05:43:37 spaha-bih ***@***.***>:

**Reply by SkewedZeppelin** [reply]
Iceraven has no added privacy features over plain Firefox.
It doesn't even remove the proprietary Google Play Services library: https://github.com/fork-maintainers/iceraven-browser/issues/463

Please see my comparison here: https://divestos.org/pages/browsers

**Comment by BaumiCoder** [comment]
Maybe [cromite](https://github.com/uazo/cromite) from @uazo can replace bromite for some of you.

@uazo You was also quite active here at bromite. Do you know something about the future development of bromite?

**Comment by 0xvino** [comment]
Cromite is cool

**Comment by hank-tu-biz** [comment]
Yes, Cromite is cool but in the meantime I found Iceraven. I like it better.

Sep 14, 2023 12:20:43 0xvino ***@***.***>:

**Reply by SkewedZeppelin** [reply]
Iceraven is not at all comparable to Bromite/Cromite

Iceraven only has ~5 features: https://github.com/fork-maintainers/iceraven-browser/issues/470#issuecomment-1119926488
Iceraven still contains proprietary Google code: https://github.com/fork-maintainers/iceraven-browser/issues/463

Iceraven is currently one release behind and vulnerable to a critical zero-day: https://www.mozilla.org/en-US/security/advisories/mfsa2023-40/
(although Bromite currently has 265 known security issues)

See also my:
- Firefox update history table: https://divestos.org/misc/ffa-dates.txt
- Chromium update history table: https://divestos.org/misc/ch-dates.txt
- Browser comparison table: https://divestos.org/pages/browsers

**Reply by 0xvino** [reply]
So it looks like Cromite is the best mobile browser for security and privacy these days ?

**Comment by hank-tu-biz** [comment]
Thanks much. I'll check it out. For a novice user like me, Iceraven is easier & more intuitive. Clearing tabs & history, returning to home screen, etc. This is very good info you've given. I'll check it out. Regards.

Sep 14, 2023 18:53:50 Tad ***@***.***>:

**Comment by BlackSparkz** [comment]
> I found a privacy browser called Iceraven. I like it better than Bromite. Also found was a FOSS app called FFUpdater. This app tracks and updates various privacy apps, including DDG, Iceraven, Tor, Brave, and Orbot plus others. It providers the latest right from GitHub.
> 
> You might check these out. They made thing easy for me.
> 
> Jul 19, 2023 05:43:37 spaha-bih ***@***.***>:
> 
> > 
> > It would be good to know at least is it the project abandoned totally or the dev is taking a longer brake.
> > 
> > Is there anything else than can replace look and feel of Bromite?
> > 
> > —
> > Reply to this email directly, view it on GitHub[https://github.com/bromite/bromite/discussions/2632#discussioncomment-6488469], or unsubscribe[https://github.com/notifications/unsubscribe-auth/BABWV5T7VD4E2YSX3LDVU7LXQ7I7RANCNFSM6AAAAAAZZSISE4].
> > You are receiving this because you commented.[Tracking image][https://github.com/notifications/beacon/BABWV5TEHD5VXK6HH3B77P3XQ7I7RA5CNFSM6AAAAAAZZSISE6WGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQAMMAZK.gif]
> > 
> 

Do you know mull 😎

**Comment by hank-tu-biz** [comment]
Bromite was abandoned. The project was adopted and rebranded & relaunched as Cromite. It seems to be under active development. The asset is available on GitHub. (I have it; I like it.) Last I checked, Cromite has not yet been added to the privacy browsers available through the FFUpdater app. Hope this helps.

Sep 26, 2023 08:45:20 Shivam hobbyist ***@***.***>:

**Reply by secuencias** [reply]
The strange thing is that I see that csagan5 has just made commits on his repositories
![IMG_0020](https://github.com/bromite/bromite/assets/138529700/1ec10410-ea48-40d3-8744-da4abe37a883)

**Reply by SkewedZeppelin** [reply]
these are automated changes

**Comment by hank-tu-biz** [comment]
commits? comments?

Sep 26, 2023 13:31:34 secuencias ***@***.***>:
